### PR TITLE
Added a few queue micro examples

### PR DESCRIPTION
**Pull request by KennethNielsen**
A have added a few queue related examples to the micro examples page.

Let me know what you think.

### CODE REVIEW

**Comment by ghcs27** [comment]
I think it is very good to expand the examples page. I like the examples you added, but the examples page doesn't seem to me to be the best place for describing `total_matches` and `number_returned`. Does this already exist somewhere? If not, I'd suggest moving your description to the `ListOfMusicInfoItems` docstring and linking from the example page to there.

**Comment by KennethNielsen** [comment]
@ghcs27 no, I think I remember thinking that it didn't feel correctly placed there, but I could quite figure out what to do with it. I think your suggestion of moving it to the ListOfMusicInfoItems docstring makes sense. I'll make the change soon. Let me know if there are any comments for grammar, spelling or formulations.

**Comment by KennethNielsen** [comment]
How about that? You think that is an appropriate level of the example?

**Comment by ghcs27** [comment]
Looks great 👍

File: soco/data_structures.py [hunk 1]
```diff
@@ -1012,7 +1012,18 @@ class DidlMusicGenre(DidlGenre):
...
+    """Abstract container class for a list of music information items.
+
+    Instances of this class are returned from queries into the music library
+    or to music services. The attributes :attr:`~total_matches` and
+    :attr:`~number_returned` are used to ascertain whether paging is required
+    in order to retrive all elements of the query. :attr:`~total_matches` is
+    the total number of results to the query and :attr:`~number_returned` is
+    the number of results actually returned. If the two differ, paging is
+    required. Paging is typically performed with the ``start`` and
+    ``max_items`` arguments to the query method.
```
Comment (ghcs27) [inline soco/data_structures.py]: Just one thing: A link to `music_library.get_music_library_information` would be good.

Comment (KennethNielsen) [inline soco/data_structures.py]: Agreed. Thanks.

**Pull request merged**